src/main.c

```diff
@@ -349,7 +349,7 @@ printUsage(ENGINE* engine) {
 
   ENGINE_printLog(engine, "  dome [options]\n");
   ENGINE_printLog(engine, "  dome [options] [--] entry_path [arguments]\n");
-  ENGINE_printLog(engine, "  dome -e | --embed [--] sourceFile [moduleName] [destinationFile]\n");
```

Comment:
The `[--]` should be kept because it is part of the usage format, even if it is optional.

Reply:
Since we are using raw argc and args the `--` is considered a valid param for a filename. So using it will cause file not found.